CONTRIBUTING.md

```diff
@@ -35,28 +39,31 @@ notes:
       url: https://community.ubuntu.com/
 ```
 
-#### Publishing
+#### Publication metadata
 
-The first half of the frontmatter contains publishing and categorization information.
+The first half of the frontmatter contains publishing and categorization information for the post.
 
-- `layout` is the page layout for Jekyll. **Do not change this**
-- `title` in this case will refer to the distribution name
-- `date` is the date of publication (addition)
+- `layout` (always `distro`) is the page layout for Jekyll
+- `title` is the distribution name
+- `date` is the date of publication of the post (i.e. addition of the distribution to this project)
+- `updated` is the date of the last update to the post (e.g. changing the version, fixing a typo, etc.)
+- `updater` is the name of the person who last updated the post
 - `logo` is the file name of the logo that you provide (minus the extension)
-- `image` is the filename of the screenshot you provide
+- `image` is the filename of the screenshot you provide (including the extension)
 - `caption` is an optional description of the screenshot
-- `tags` are an array of tags for the post, these should be limited to things like the desktop environments it uses, it's primary purpose (e.g. server, desktop, enterprise), a one-word estimation of it's difficulty
+- `tags` is an array of tags for the post. These should be limited to things like its primary purpose (e.g. `server`, `desktop`, `enterprise`)
+- `category` is a one-word estimation of the distribution's approachability (`beginner`, `intermediate` or `advanced`)
 
-#### Notes
+#### Distribution metadata
 
-The `notes` section contains pertinent info about a distribution that show's up in the 'Notes' section on each post.
+The `notes` section contains pertinent info about the distribution itself that shows up in the "Notes" section on each post.
```

Comment:
```suggestion
The `notes` section contains pertinent info about the distribution itself that shows up in the “Notes” section on each post.
```

Reply:
I generally prefer using straight quotes in markdown source, since those are easier to produce in most keyboard layouts, and renderers can often auto-convert to smart quotes. Can you explain your reasoning for preferring the latter?

Reply:
If the renderer does it, then fine yeah. If they don't, then this is needed. AFAIK Jekyll does not do it out-of-the box so, this is more practical and less theory.

The quotes I use are just the correct typographical quotes. If you are into that stuff (or ever wrote anything with LaTeX :wink:) you see these typographical errors everywhere. :upside_down_face:

Reply:
Heheh gotcha. However, this file is not rendered on the site, so Jekyll's rendering is not relevant, but rather what GitHub uses to render markdown here, as we browse the source code. [This](https://github.com/waldyrious/computefreely/blob/main/CONTRIBUTING.md) is what it looks like.

Do you feel the lack of typographical quotes in that page is too conspicuous? I would prefer to not include them in the source code as that makes it harder to edit (I, for once, have to copy existing instances because I don't have a convenient way to input them — and indeed [most people don't](https://en.wikipedia.org/wiki/Quotation_mark#Keyboard_layouts)), and the straight quotes in the rendered text don't bother me.

That said, if you do feel strongly about this, I'd be open to adding them anyway as my objection is relatively light.

Reply:
Yeah GitHub does not special rendering. I know it's unpractical, you have to pay attention etc. but it's just proper typesetting. Most people don't care, if you read newspaper articles you will see they (obviously should) care etc.

But as we are in a Linux project some tips/FYI:
* for such special characters [the compound key is actually very useful](https://askubuntu.com/questions/70784/how-can-i-enable-compose-key)
* for these quotes, I at least in a German QUERTZ layout do have (by default) a third special characters at the <kbd>v</kbd> („), <kbd>b</kbd> (“), <kbd>n</kbd> (”) keys:
   ![image](https://github.com/waldyrious/computefreely/assets/11966684/fb6a91a9-8930-4259-92dd-ab8f34e68d12)
   I am personally quite accustomed to using these for such inputs.

Reply:
Huh. I just checked and indeed in my Portuguese QWERTY layout, `“` and `”` are also in the <kbd>V</kbd> and <kbd>B</kbd> keys:

![image](https://github.com/waldyrious/computefreely/assets/478237/36eab4a5-d877-4b67-95a8-fc946b12a37a)

So I suppose it's less of a chore to keep using these than I assumed. I'll go ahead and integrate your suggestions :)